Add metric script to scrape the rls-*-bug-tasks reports

OddBloke opened the pull request.

powersj commented:
@murraybd can you review this one as well?

murraybd commented on metrics/rls_bug_tasks.py:
```diff
@@ -0,0 +1,61 @@
+REPORT_PARENT = 'http://reqorts.qa.ubuntu.com/reports/rls-mgr/'
+REPORT_URL_PATTERN = (
+    REPORT_PARENT + 'rls-{release_prefix}-{status}-bug-tasks.html')
+STATUSES = ['incoming', 'tracking']
```
I think using the word STATUSES here and more so later on with "status in STATUSES" might be confused with Launchpad bug task statuses which is not what these are, maybe using TAGS would be less confusing?

murraybd commented on metrics/rls_bug_tasks.py:
```diff
@@ -0,0 +1,61 @@
+    response = requests.get(REPORT_URL_PATTERN.format(
+        release_prefix=release_prefix, status=status))
+    return dict(re.findall(r'<span id="(.+)-total">(\d+)</span>',
+                           response.text))
```
While the reports haven't changed in forever what will happen if this span id is not found? Should we guard against the report being broken?

murraybd commented on metrics/rls_bug_tasks.py:
```diff
@@ -0,0 +1,61 @@
+        print('Pushing data...')
+        registry = CollectorRegistry()
+        gauge = Gauge(
+            'rls_bug_tasks', '', ['status', 'team_name'],
```
From what I've seen a lot of our gauges (the ones we've created) have had "foundations_" as the first word, while I realize these would be useful for other teams like server, I wonder if we shouldn't also precede it with foundations_ to distinguish it from gauges they've made. Additionally, renaming the script itself to start with foundations_ might make it easier to track team ownership of the code.

powersj replied:
+1 here, namespace this with something. rls, aka release bug tasks is a little too generic.

OddBloke replied:
"foundations_" wouldn't be an appropriate prefix, because the metric will have values for more than just the Foundations team. I'm open to suggestions. Perhaps "platform_"?

murraybd replied:
Let's use "distro_" for metrics that are useful for people outside the foundations team and would also be interesting to community members.

OddBloke commented:
@murraybd I think I've addressed all comments except the inclusion of a comment about JSON files; could you either point me at those (or +1 this and then propose a merge yourself)?

murraybd commented:
Lets us "distro_" instead of "platform_" as distro feels more community while platform seems more like a Canonicalism and the rls tag information could be useful to the Ubuntu community.  After that change feel free to merge this, thanks!
